runtests.py

```diff
@@ -52,7 +52,7 @@
         ],
         # Load models directly to pick up test-only models
         # See: http://stackoverflow.com/a/25267435/347942
-        MIGRATION_MODULES={'treenav': 'treenav.migrations_not_run_during_tests'},
+        MIGRATION_MODULES={'treenav': None},
```

Comment:
Does the comment still apply?

Reply:
Yes, the hack just needs updating for newer Django per the same stackoverflow discussion.